### PR TITLE
Added missing tolerances for CompressEvents in AlignAndFocusPowderFromFiles

### PR DESCRIPTION
**Pull request by marshallmcdonnell**
**Description of work.**

Added the tolerances `CompressTolerance` and `CompressWallClockTolerance` inputs from `AlignAndFocusPowderFromFiles` to the child algorithm calls to `CompressEvents` (int `AlignAndFocusPowderFromFiles`, not just the `AlignAndFocusPowder` algorithm) as the necessary `Tolerance` and `WallClockTolerance` arguments, respectively.

**To test:**

1) Set logging to `information` level.

2) Call `AlignAndFocusPowderFromFiles` **with at least two files to load** and `CompressTolerance` = 0. Will notice that `CompressEvents` is no longer called. 

3) Also, do with `CompressTolerance` > 0. and != 1e-5 and will see that `CompressEvents` called with the set value (not just the default).

4) Similarly,  test with the `CompressWallClockTolerance` argument set and see that its value is now set in `CompressEvents` as well.

Fixes *There is no associated issue.*

Release note added for v3.14.0

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Framework/PythonInterface/plugins/algorithms/AlignAndFocusPowderFromFiles.py [hunk 1]
```diff
@@ -128,7 +128,8 @@ def __getAlignAndFocusArgs(self):
         args = {}
         for name in PROPS_FOR_ALIGN:
             prop = self.getProperty(name)
-            if name == 'PreserveEvents' or not prop.isDefault:
+            name_list = ['PreserveEvents', 'CompressTolerance', 'CompressWallClockTolerance']
```
Comment (peterfpeterson) [inline Framework/PythonInterface/plugins/algorithms/AlignAndFocusPowderFromFiles.py]: This should also copy over `CompressStartTime`

**Review by peterfpeterson — CHANGES_REQUESTED** [review]
You should add something in the release notes about this bugfix

**Review by peterfpeterson — DISMISSED** [review]
LGTM

**Review by peterfpeterson — CHANGES_REQUESTED** [review]
Still need to pass `CompressStartTime` to `CompressEvents`

**Review by peterfpeterson — APPROVED** [review]

**Review by soininen — CHANGES_REQUESTED** [review]
I cannot find any unit tests for this algorithm. Would it be time to create some?

**Comment by soininen** [comment]
If this needs to get merged in a timely manner, I am OK if you open an issue for tests and work on them later.

**Comment by marshallmcdonnell** [comment]
@soininen Thank you! See issue #23888 for [AlignAndFocusPowder](http://docs.mantidproject.org/nightly/algorithms/AlignAndFocusPowder-v1.html) and #24386 for [AlignAndFocusPowderFromFiles](http://docs.mantidproject.org/nightly/algorithms/AlignAndFocusPowderFromFiles-v1.html).

**Review by soininen — APPROVED** [review]

**Pull request merged**